### PR TITLE
Adding pub and sub to sample

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs [hunk 1]
```diff
@@ -2,13 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections;
```
Comment (vwxyzh) [inline samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs]: remove?

File: samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs [hunk 2]
```diff
@@ -29,23 +34,88 @@ static async Task Main(string[] args)
                 {
                     case Mode.Broadcast:
                         await proxy.Invoke("BroadcastMessage", currentUser, input);
-
                         break;
                     case Mode.Echo:
                         await proxy.Invoke("echo", input);
                         break;
+                    case Mode.Pub:
```
Comment (vwxyzh) [inline samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs]: group broadcast?

File: samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs [hunk 3]
```diff
@@ -29,23 +34,88 @@ static async Task Main(string[] args)
...
+                                groupName = args[1];
+                            }
+                            for (int i = 0; i < MaxMessageCount; i++)
+                            {
+                                await proxy.Invoke("publish", groupName, input, i);
+                                await Task.Delay(1000);
+                            }
+                            break;
+                        }
+                    case Mode.Sub:
```
Comment (vwxyzh) [inline samples/AspNet.ChatSample/AspNet.ChatSample.CSharpClient/Program.cs]: join group? and no leave group?

**Comment by vicancy** [comment]
close for better solution later

**Pull request closed without merging**